src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -1407,7 +1406,7 @@ public void moveStudyNode(UUID studyUuid, UUID nodeToMoveUuid, UUID referenceNod
         List<NodeEntity> oldChildren = null;
         checkStudyContainsNode(studyUuid, nodeToMoveUuid);
         checkStudyContainsNode(studyUuid, referenceNodeUuid);
-        boolean shouldInvalidateChildren = !EMPTY_ARRAY.equals(networkModificationTreeService.getNetworkModifications(nodeToMoveUuid));
+        boolean shouldInvalidateChildren = networkModificationTreeService.getNetworkModificationsCount(nodeToMoveUuid, false) > 0;
```

Comment:
Add a method in the `networkModificationTreeService:hasModifications(noduid, stahed)` ?

Reply:
Done